api/offers.py

```diff
@@ -31,7 +31,7 @@ def offers_response(response_dict):
 
 def filterOffersByTime( currency_type , time_seconds):
     #filter by currency
-    currency = ('1' if currency_type == 'MSC' else '2')
+    currency = ('1' if currency_type == 'OMNI' else '2')
```

Comment:
@Nevtep  Similar to rest let's try to respect previous 'requests' to preserve backwards compatibility     currency= ('1' if (currency_type == 'OMNI' or currency_type == 'MSC') else '2')